eclass/kernel-install.eclass

```diff
@@ -547,6 +547,20 @@ kernel-install_pkg_pretend() {
 			elog "    emerge --config ${CATEGORY}/${PN}:${SLOT}"
 		fi
 	fi
+
+	if ! use initramfs && ! has_version "=${CATEGORY}/${PN}[-initramfs]"; then
```

Comment:
That's not a valid atom.

Reply:
Oops, that `=` is left over from when it was `PF` instead of `PN`.